Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/CreateInviteOwnerIntroViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  AddGeneratedKeyAsOwnerIntroViewController.swift
+//  Multisig
+//
+//  Created by Moaaz on 6/9/22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class CreateInviteOwnerIntroViewController: UIViewController, UIAdaptivePresentationControllerDelegate {
+
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var shareButton: UIButton!
+    @IBOutlet weak var notThisTimeButton: UIButton!
+
+    var onShare: (() -> ())?
+
+    var onSkip: (() -> ())?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        presentationController?.delegate = self
+        titleLabel.setStyle(.primary)
+        descriptionLabel.setStyle(.secondary)
+        shareButton.setText("Share link", .filled)
+        notThisTimeButton.setText("Not this time", .plain)
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        Tracker.trackEvent(.addAsOwnerIntro)
+    }
+
+    // Called when user swipes down the modal screen
+    func presentationControllerDidDismiss(_ presentationController: UIPresentationController) {
```

Comment:
can you check that this actually called? I think the swipe-to-dismiss is handled in another place (in navigation controller), but not sure.

When I tried to swipe down but then moved the finger up, the screen still swiped down, however I expected it to stay.

Reply:
It turns out that it's general issue that we have unfortunately
Let's fix it separately